lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -336,15 +338,30 @@ ParseResult FIRParser::parseToken(FIRToken::Kind expectedToken,
   return emitError(message);
 }
 
-/// Parse a list of elements, terminated with an arbitrary token.
+/// Parse a comma-separated list of elements, terminated with an arbitrary
+/// token.
```

Comment:
Now that there are explicit commas, isn't there an existing delimiter separated list parse function we can just use?

Reply:
If someone knows of such a function, I can switch it out.
Considering the whole parser is hand-written, I wasn't expecting there to be much in the way of parsing utilities.